react/react1/week3/my-todo-list/src/components/TodoItem.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const Text = styled.p`
+  text-decoration: ${(props) => (props.checked ? 'line-through' : 'none')};
+`;
+
+const TodoItem = (props) => {
+  const { checked, description, deadline } = props;
```

Comment:
Istead of destructuring here do it inside the function paramaters as you have done with the others

Reply:
Hello
Thanks for the review.
I want to understand for me which way is better. If we plan to increase the application in the future, isn't it a better way to use destructuring from the start?

Reply:
Hello Yuliia. A general comment to this is that in react we generally don't want a component to have too many props as components become hard to maintain when they are starting to do too much and manage too much. My rule of thumb is that a component should not have much more than 3 props. Whenever they start to exceed that i start thinking about how to restructure the components and how they work together. This is also the reason it is better to destructure inside the function as it is easier to see which props the component has and it makes it harder to just pass in 10 different props, which we always should avoid. You can think of a component as a regular function. In programming we never want to have a function with more than 3 props because the functions are doing too much by then. There are ofcourse exceptions to the rule and i don't always do things this way. 

I know this was a bit of a long comment, but i hope it helps, otherwise just send more questions my way :D